app/waf/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# build with:
+#   docker build . -t f5devcentral/f5-wafaas-openstack:latest
+
+# upload image with (authorized user only)
+#   docker push f5devcentral/f5-wafaas-openstack:latest
+
+FROM node:11.6.0
+
+LABEL maintainer="F5 Networks Inc. Corp."
+EXPOSE 3000/tcp
```

Comment:
Is it configurable?

Reply:
The port is only a convention, only visible in the running container. 
I think there's no need to make it configurable.